specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -44,6 +44,18 @@ model InputDataset extends InputData {
   id: string;
 }
 
+@doc("Dataset as source for evaluation.")
+model RawDataset extends InputData {
+  type: "dataset";
```

Comment:
type is polymorphic. The type "dataset" is already used to indicate dataset asset. Can we please use another type here?